UI/src/view/point_asset/massTransitStopForm.js

```diff
@@ -924,12 +1011,96 @@
               'Olet siirtämässä pysäkin ELYn ylläpitoon! Huomioithan, että osa pysäkin varustetiedoista saattaa kadota tallennuksen yhteydessä.',
               {type: 'alert'});
         }
+
+        if (property.publicId === 'palvelu') {
+          hideOrShowTarkenne();
+          updateViranomaisdataaValue();
+
+          pointAssetToSave = true;
+        }
+
       });
 
       eventbus.on('terminalBusStop:selected', function(value) {
         isTerminalBusStop = value;
       });
 
+
+
+      function isAuthorityData(selectedServiceType) {
+        return !(selectedServiceType === 10 || selectedServiceType === 17);
+      }
+
+
+      function createNewServiceDropDown()
+      {
+        var enumVals = _.find(enumeratedPropertyValues, function(choice){
+                           return choice.publicId === 'palvelu';
+                      }).values;
+
+        var result = $('<div />').addClass('form-group new-service');
+
+        result = result.append( $('<select />').addClass('form-control select').change(newServiceSelectOnChange)
+                        .append($('<option />').text('Lisää uusi rajoitus').addClass('empty').attr('disabled', true).attr('selected', true))
```

Comment:
change Lisää uusi rajoitus to Lisää uusi palvelu

Reply:
Done